db-schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE "chat_message" (
+    "chat_id" uuid REFERENCES "item" ("id") ON DELETE CASCADE,          -- id of item to which chat is attached
```

Comment:
Do we not want to call this `item_id`, since it references `item`?

Reply:
I asked myself the same question. I decided on using `chat_id` instead to future-proof the schema, for instance if one day we decide to implement chat for things other than items.